### DISCUSSION TITLE
First discussion Eva

### DISCUSSION THREAD

**Discussion by Anton-Latukha** [haskell-nix/hnix#771] [General]
Lol wassup :t-rex:

Let's see how this works.

**Comment by layus** [comment]
Oh no ! You could not resist clicking the "new feature" popup :joy:

I guess this is for PRs with no code, or issues with no fixes :-). Might be useful though for taking decisions and evaluating options.

**Reply by Anton-Latukha** [reply]
Actually, I observed the lack of this and was kind-of waiting for GH to ship some communication chats.

Reports require a right-away proper topic starter attack to unwrap the topic.

Sometimes the topic is too complex, and has a lot of ways to go about it/solve it, discussions are for those type of cases.

Moreover, they free-form and can be mish-mash, and lines of thought in them can be taken and reports/patches can be formalized/programmed.

Also notice that every message has a subthread, so here the lines of discussions are more structured.
